recipes-devtools/gie/tensorrt-plugins_8.0.1-1.bb

```diff
@@ -0,0 +1,52 @@
+DESCRIPTION = "NVIDIA TensorRT Plugins for deep learning"
+HOMEPAGE = "http://developer.nvidia.com/tensorrt"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=a55f7353e48c6734c10531db0b040177"
+
+inherit cuda cmake 
+
+OECMAKE_GENERATOR = "Unix Makefiles"
+
+SRC_REPO = "github.com/NVIDIA/TensorRT.git;protocol=https"
+SRCBRANCH = "master"
+SRC_URI = "gitsm://${SRC_REPO};branch=${SRCBRANCH} \
+    file://0001-CMakeLists.txt-fix-cross-compilation-issues.patch \
+"
+SRCREV = "eb5de99b523c76c2f3ae997855ad86d3a1e86a31"
+
+S = "${WORKDIR}/git"
+B = "${S}"
+
+DEPENDS += "zlib libcublas cudnn cuda-cudart cuda-nvrtc protobuf protobuf-native tensorrt-core"
```

Comment:
The dependency on protobuf, which lives in meta-oe, means that this recipe cannot be located here.  It will have to be moved under `external/openembedded-layer`.

Alternatively, we could revert to using the included protobuf sources and remove this dependency, although I'm not crazy about that idea.

Reply:
I moved the recipe under `external/openembedded-layer` for now. Agree, I prefer to use the internal `Protobuf`  to avoid depending on `protobuf` and  `protobuf-native`